content/chainguard/chainguard-images/working-with-images/cve_visualizations/index.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "Using CVE Visualizations"
+linktitle: "CVE Visualizations"
+type: "article"
+description: "Getting started with the CVE Visualization feature."
+date: 2024-12-04T11:07:52+02:00
+lastmod: 2024-12-04T11:07:52+02:00
+draft: false
+tags: ["CONCEPTUAL", "CHAINGUARD IMAGES", "PRODUCT"]
+images: []
+menu:
+  docs:
+    parent: "chainguard-images"
+weight: 020
+toc: true
+---
+
+The CVE Visualizations feature in the Chainguard Console helps teams measure the engineering,
+security and economic benefits gained from using Chainguard Images. The primary feature is a
```

Comment:
We'll also be adding a separate tab in the Reports section for Beta with historical CVE remediation )showing the # of resolved CVEs over time per image). Maybe reword to something like "The feature provides self-service reporting for CVE comparisons counts between Chainguard Images and popular open-source alternatives, as well as historical CVE remediation metrics for your entitled Chainguard Images"

Reply:
Tried to reword a bit. "self service" seems redundant in this context and the metrics seem to be for all images now (not just entitlements)